sdk/servicebus/service-bus/test/invalidParameters.spec.ts

```diff
@@ -668,7 +668,7 @@ describe("invalid parameters", () => {
         TestClientType.PartitionedQueue
       );
 
-      //const clients = await getSenderReceiverClients(TestClientType.PartitionedQueue, "peekLock");
+      // const clients = await getSenderReceiverClients(TestClientType.PartitionedQueue, "peekLock");
```

Comment:
Oh, thank goodness this finally got fixed. ;)

(these are great btw, thanks for doing the linting)